src/index.css

```
@@ -2,6 +2,11 @@
   --color-blue: #0070f0;
   --color-light-blue: #32b0fd;
   --margin-lr: 14px;
+  padding-top: constant(safe-area-inset-top);
```

Biome (2.1.2)

[error] Unexpected unknown function: constant

Use a known function instead.
See MDN web docs for more details.

(lint/correctness/noUnknownFunction)